### PR TITLE
Make modified not an observable

### PR DESCRIPTION
**Pull request by Ghalko**
Used two different places, one as an observable, the other not. Made not an observable.

### CODE REVIEW

File: website/static/js/comment.js [hunk 1]
```diff
@@ -302,7 +302,7 @@ CommentModel.prototype.submitEdit = function(data, event) {
         self.content(response.content);
         self.dateModified(response.dateModified);
         self.editing(false);
-        self.modified(true);
+        self.modified = true;
```
Comment (sloria) [inline website/static/js/comment.js]: `self.modified` is only used by `editVerb`, which appears to be unused. Perhaps we just remove `modified` and `editVerb`. @Ghalko can you see if this is do-able?

**Comment by Ghalko** [comment]
Can do.

**Comment by Ghalko** [comment]
"modified" is used. It is used in the template to make an asterix appear. Should I try to remove this as per @sloria talk yesterday and move it into the js? I think this is okay because it isn't the very odd comment if statements.

**Comment by Ghalko** [comment]
The only place I can think that editVerb is used would be Emails, but I have those set up and I can't see any verbs "edited" or "posted" in the old Emails, and certainly not after removing the editVerb.

**Comment by sloria** [comment]
Oh ok. I didn't see that usage of `modified`.

I think what you have here is correct.

**Pull request merged**